--- a/lib/tsnnl.c
+++ b/lib/tsnnl.c
@@ -1279,6 +1279,9 @@
 	if (enable == 1)
 		tsn_send_cmd_append_attr(msg, TSN_QBV_ATTR_ENABLE, &enable, 0);
 
+	if (adminconf->maxsdu)
+		tsn_send_cmd_append_attr(msg, TSN_QBV_ATTR_MAXSDU, &(adminconf->maxsdu), sizeof(adminconf->maxsdu));
+
 	if (adminconf->config_change)
 		tsn_send_cmd_append_attr(msg, TSN_QBV_ATTR_CONFIGCHANGE, &(adminconf->config_change), 0);
 
